resources/language/resource.language.es_es/strings.po

```diff
@@ -292,11 +294,11 @@ msgstr "Vista para los exportados"
 
 msgctxt "#30075"
 msgid "Ask to skip intro and recap"
-msgstr "Preguntar si omitir intro y resumen"
+msgstr "Preguntar si omitir la introducción y resumen"
 
```

Comment:
DON'T AGREE, because I think the shorter the better for the control containing the text.

Reply:
AGREED